### ISSUE TITLE
2024 -2025 Mobilisation: Australian Wildlife Conservancy (Main)

### ISSUE THREAD

**Issue by rosemaryjoconnor** [AtlasOfLivingAustralia/data-management#1066]

**Comment by rosemaryjoconnor** [comment]
**02/10/2024**

- Startup meeting held today.


**Summary**

- 310,000 records approx to be delivered
- approx 50% of data has been transferred to new DB- final design is still in progress
- remainder to be digitised as part of mobilisation
- Wanting to use this as an opportunity to learn an understand DwC so they can make this more their BAU and support ongoing data provision 
- Sensitive data - some of the data is from fenced feral-free areas - including Scotia Wildlife sanctuary. Discussions ongoing as to how they want to handle sensitive data and indigenous locations. 
- Some data is already in Bionet - this will likely be removed from Bionet and superseded by the new data as Bionet doesn't include data for effort and other parameters.
- NSW Bionet as an events dataset, but data rights holder and institution code is Australian wildlife Conservancy. [See here.](https://biocache.ala.org.au/occurrences/search?q=lsid%3Ahttps%3A%2F%2Fbiodiversity.org.au%2Fafd%2Ftaxa%2Fd4fe643e-881f-49de-a8c9-628d5bce2e24&qualityProfile=ALA&fq=state%3A%22New%20South%20Wales%22&qc=-_nest_parent_%3A*&lat=-33.155624&lon=141.15446&radius=10.4&fq=data_resource_uid%3A%22dr368%22) 


**Note:** may need to make sure field does not indicate these animals as truly wild - could be misleading to list them as wild as they are considered extinct in NSW and are in a fenced area. They might have an update to this data perhaps.

AWC to contact late Oct re sample data and new DB analyst

**Comment by rosemaryjoconnor** [comment]
**31/10/2024**

Emailed CM to arrange mtg

**Comment by rosemaryjoconnor** [comment]
**08/11/2024**
Good mtg!

- Possibly using events for dataset - occurrences and effort to be provided. 
- Currently effort is linked to a survey/event -> location rather than occurrence
- Camera trap data, recorded every 5 mins - what do we need
- CM to provide a sample extract of occurrence and event/effort data
- AWC to determine how to deal with data sensitivities
- Bionet data will likely stay due to statutory requirements. But records to ALA will have more data associated

**Comment by rosemaryjoconnor** [comment]
**09/11/2024**
Sample data provided. 

- Structured well for events - all survey based
- Camera trap data - Raj work will help with this
- Emailed to request some sample camera trap data
- Obfuscate locations until sensitive locations defined
- Create data mappings and send

**Comment by rosemaryjoconnor** [comment]
**17/01/2025**

PN following up re sensitive data

**Comment by rosemaryjoconnor** [comment]
**25/03/2025**

- Mtg re sensitive data scheduled for 27/03/2025
- AWC sample data to events - in progress

**Comment by rosemaryjoconnor** [comment]
**01/04/2025**

Excellent meeting with CM - over 2 hours

- CM to look at measurements and facts and effort data
- AWC to provide separate csv files with occurrences/location/event data, effort data, measurements or facts 

**To do:**
- [ ] Provide DWC/ALA terms and field name mappings
- [ ] Restructure of events to be closer to AWC survey methodology
- [ ] Include institution in occurrences
- [ ] Some re-mapping to be done

**Comment by rosemaryjoconnor** [comment]
**11/04/2025**
Multiple options for events
Currently in databox - sites/surveys display and issue in events. Raised for update see:

**Comment by rosemaryjoconnor** [comment]
**07/05/2025**
Emailed CM with mappings, sample structure etc...
Meeting set for 12/05/2025

**12/05/2025**
Mtg went well. Updated mappings document sent.

**Comment by rosemaryjoconnor** [comment]
**24/07/2025**
Files provided by AWC 27/05/2025

- AWC_eventSiteVisit.xlsx.csv
- AWC_eventSurvey.csv
- AWC_occurrenceRecords.csv

Event files contain site location and effort information so have not been included here. 
Occurrence file needed to be saved as utf-8

`dr23130` - AWC occurrences only with 1km obfuscation
Also outputs raw occurrence file for RASD?

Do we need to provide complete occurrence and events files for RASD?

**Comment by rosemaryjoconnor** [comment]
**23/12/2025**

**Databox**
- [ ] Data Provider: `dp5223`: https://collections-test.ala.org.au/dataProvider/show/dp5223
- [ ] Delete Institution: `in427`: https://collections-test.ala.org.au/institution/show/in427
- [ ] DataResource: `dr23130` : https://collections-test.ala.org.au/dataResource/show/dr23130
- [ ] DataResource: `dr499` - update as records not events and replace with data as per dr23130
- [ ] Delete `dr23130` once all loaded in Production

**Prod**
- [ ] Data Provider: `dp5217`: https://collections.ala.org.au/dataProvider/show/dp5217
- [ ] Delete Institution: `in427`: https://collections.ala.org.au/institution/show/in427
- [ ] DataResource: `dr499` - preingest/load data
- [ ] DataResource; `dr499` set to public, sync with GBIF

**Comment by rosemaryjoconnor** [comment]
**24/12/2025**
Updates to data:

- renamed observationType to basisOfRecord
- changed all basisOfRecord values 'camera image' to 'machineObservation'

**Comment by rosemaryjoconnor** [comment]
**06/01/2026**
Databox updated. Waiting on response
